test/influx_udp_test.erl

```diff
@@ -59,3 +72,13 @@ write_point_to_pool_t_({_UDP1, UDP2}) ->
   [
     ?_assertEqual(<<"test_pool">>, gen_server:call(UDP2, msg))
   ].
+
+write_influx_point_t_({UDP1, _UDP2}) ->
+  influx_udp:write([
+    #{ measurement => <<"test">>, fields => #{ val => 1}, time => 1},
+    #{ measurement => <<"test2">>, fields => #{ val => 2}, time => 2}
+  ]),
+  timer:sleep(1),
+  [
+    ?_assertEqual(<<"test val=1 1\ntest2 val=2 2\n">>, gen_server:call(UDP1, msg))
```

Comment:
Line 83 is too long:     ?_assertEqual(<<"test val=1 1\ntest2 val=2 2\n">>, gen_server:call(UDP1, msg)).